### ISSUE TITLE
Add option to turn on or off solar panels

### ISSUE THREAD

**Issue by Sicos1977** [PiotrMachowski/Home-Assistant-custom-components-GNE-PV-Monitoring#4]
Hi,

Could you add an option to turn on or off the solar panels? This should be possible through the API.

Sending this payload to the API turns everything on --> {stationId: "<station id>", ids: "FFFFFFFF", status: "0"}
And this to turn everything off --> {stationId: "<station id>", ids: "FFFFFFFF", status: "1"}

**Comment by preussal** [comment]
That would be brilliant, then you could also switch off individual panels to prevent overvoltage on the string

**Comment by vonicoza-create** [comment]
yes, please, that would be great. willing also to send some fee for Your help :-)
